src/pages/TutorProfilePage.tsx

```diff
@@ -0,0 +1,766 @@
+                                                                selectedOption.map(
+                                                                    (item) => item.value
+                                                                )
+                                                            ); //updates selected options in tutorRequest
+                                                        }}
+                                                    />
+                                                )}
+                                            </Field>
+                                        </FormControl>
+                                    </Box>
+                                </Box>
+
+                                <Box>
+                                    {/* MATH*/}
+                                    <Box maxW="35em" p="3px">
+                                        <FormControl>
+                                            <Field name="MathSubject">
+                                                {({ field, form }) => (
+                                                    <MultiSelect
+                                                        {...field}
+                                                        options={mathSubjectOptions}
+                                                        value={form.values.MathSubject.map(
+                                                            (item) => {
+                                                                return {
+                                                                    label: item,
+                                                                    value: item,
+                                                                };
+                                                            }
+                                                        )}
+                                                        label="Math"
+                                                        onChange={(selectedOption: Option[]) => {
+                                                            form.setFieldValue(
+                                                                field.name,
+                                                                selectedOption.map(
+                                                                    (item) => item.value
+                                                                )
+                                                            ); //updates selected options in tutorRequest
+                                                        }}
+                                                    />
+                                                )}
+                                            </Field>
+                                        </FormControl>
+                                    </Box>
+                                    {/* GRADES*/}
```

Comment:
Exchange  fields:  "Grades" and "Foreign languages" for better flow

Reply:
Yep!